src/Validator/TableExistsHandler.php

```diff
@@ -28,7 +28,11 @@ public function validate(mixed $value, object $rule, ValidationContext $context)
 
         $result = new Result();
 
-        $tableSchema = $this->connection->getTableSchema($value);
+        try {
+            $tableSchema = $this->connection->getTableSchema($value);
+        } catch (\Yiisoft\Db\Exception\Exception) {
+            $tableSchema = null;
```

Comment:
I think, better add validation error with exception message instead of suppressing.

Reply:
added